ts/.gitignore

```diff
@@ -3,6 +3,7 @@
 /build
 /node_modules
 tsconfig.paths.json
+/src/generated/
```

Comment:
code-generation should not be gitignored and instead available for all other users, so they don't go thru hoops of setting up environment and generation code

Reply:
I think 2 things here:
1. If smb wants to contribute, he needs to generate files from proto. Moreover, I think we must do this in gha and fail PR if this doesn’t work. Just to ensure that TS files are in sync with proto files
2. We need clear documentation on how to start the project from scratch for development.